charts/flink-job/templates/flinkjobcluster.yaml

```diff
@@ -53,7 +53,19 @@ spec:
       {{- toYaml . | nindent 6 }}
       {{- end }}
     {{- end }}
-  # volumeClaimTemplates, initContainers, nodeSelector, tolerations, sideCars not exposed
+    # volumeClaimTemplates, initContainers, sideCars not exposed
+    {{- if .Values.nodeSelector }}
+    nodeSelector:
+      {{- with .Values.nodeSelector }}
+      {{- toYaml . | nindent 6 }}
+      {{- end }}
+    {{- end }}
+    {{- if .Values.tolerations }}
```

Comment:
@eugenend You don't have to change this.. but in the interest of teaching you some helm tricks.. this can be shortened.

Long:
```yaml
    {{- if .Values.tolerations }}
    tolerations:
      {{- with .Values.tolerations }}
      {{- toYaml . | nindent 6 }}
      {{- end }}
    {{- end }}
```

Short:
```yaml
    {{- with .Values.tolerations }}
    tolerations:
      {{- toYaml . | nindent 6 }}
    {{- end }}
````

Reply:
This is a good trick, thank you!